nautobot/extras/context_managers.py

```diff
@@ -16,14 +17,14 @@ class ChangeContext:
     one will be generated to relate any changes to this transaction. Convenience
     classes are provided for each context.
 
-    :param user: User object
+    :param request: WSGIRequest object
     :param context: Context of the transaction, must match a choice in nautobot.extras.choices.ObjectChangeEventContextChoices
     :param context_detail: Optional extra details about the transaction (ex: the plugin name that initiated the change)
-    :param id: Optional uuid object to uniquely identify the transaction
+    :param id: Optional uuid object to uniquely identify the transaction. One will be generated if not supplied
```

Comment:
My only gripe now that I'm looking at this again is the name of the `id` parameter since it shadows the build-in `id()` function. Outside of that, this is good to go.

Reply:
I renamed that attribute to `change_id`

Reply:
Uh oh forgot to update the docstring?